src/react-apps/applications/runtime/src/components/base/FileUploadComponent.tsx

```diff
@@ -117,11 +126,10 @@ export class FileUploadComponentClass
     this.setState({
       attachments: newList,
     });
-    FormFillerActionDispatchers.deleteAttachment(attachmentToDelete, fileType, this.props.id);
-
+    FormFileUploadDispatcher.deleteAttachment(attachmentToDelete, fileType, this.props.id);
   }
 
-  public getComponentValidations = (): IComponentValidations => {
+  public getComponentValidations = (): any => {
```

Comment:
Why is IComponentValidations replaced with any?

Reply:
I'm guessing we did not have the interface when the component was copied over, fixed now ;)